easybuild/tools/github.py

```diff
@@ -243,10 +243,32 @@ def github_api_get_request(request_f, github_user=None, token=None, **kwargs):
     url = request_f(RestClient(GITHUB_API_URL, username=github_user, token=token))
 
     try:
-        status, data = url.get(**kwargs)
+        status, headers = url.head(**kwargs) 
     except socket.gaierror as err:
         _log.warning("Error occurred while performing get request: %s", err)
-        status, data = 0, None
+        status, headers = 0, None
+
+    links = headers.getheader('Link')
```

Comment:
@migueldiascosta This will fail miserably if `headers` was set to `None` above...

Reply:
@boegel tfw you realise you created two branches at different times to work on the same thing and pushed the older one...

Reply:
hehe